### ISSUE TITLE
Problems accessing data 

### ISSUE THREAD

**Issue by jessiebullock** [lucasmation/microdadosBrasil#100]
When I try to download one of the numerous datasets I get a strange error message where it doesn't list one of the "following" it may be below, it stops at the warning pasted below:

`> download_sourceData("CENSO", 2000, unzip = T)
Error in download_sourceData("CENSO", 2000, unzip = T) :
 Invalid dataset. Must be one of the following:  `

I have downloaded the exdata folder from git but unsure if there is something I missed in downloading or reading the files? Is there some working directory I should have set (right now set to the folder the exdata folder is in) or data I should have previously downloaded? Unsure why this isn't working. Thanks!

**Comment by lucasmation** [comment]
@jessiebullock, this looks like a bug on our part. We will fix that soon
@nicolassoarespinto, maybe this was caused by the latest overhaul of the package (#97) . We need run the tests for all datasets...

**Comment by nicolassoarespinto** [comment]
@jessiebullock Just fixed it in #102 . Please see if it works for you. 

 Also, you should set the working directory to the folder you want the data do be stored in or use the `dest` argument.

```
setwd(".../microdata")
download_sourceData("CENSO", 2000, unzip = T)
```
or
`
download_sourceData("CENSO", 2000, unzip = T, dest = ".../microdata")`

@lucasmation  working on that as soon as possible.

**Comment by jessiebullock** [comment]
Its all working now. Thanks!
